src/AbsoluteHack/templates/AbsoluteHackInit/index.html

```diff
@@ -5,18 +5,18 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no" />
 	<title><%= htmlWebpackPlugin.options.title %></title>
-
-	<%= require('./_meta.ejs')({ data: htmlWebpackPlugin.options }) %>
-	<%= require('./_favicons.ejs')() %>
+	{% load static %}
```

Comment:
Revert all the changes made in this file. Just modify the file paths only, according to your project structure.